src/components/GameCard.tsx

```diff
@@ -9,17 +9,71 @@ interface GameCardProps {
     gameTitle: string;
     gameImage: any;
     gameDescription: string;
+    gamePlatforms: Array<string>;
+    gameGenres: Array<string>;
```

Comment:
Are we going to define enums for platforms and genres to standardize them or will these be fully freeform?

Reply:
Yes! I was starting to work on this, but it was too much overhead work for now. Just created two tickets (put them for milestone 4 since that's going to be the start of cleaning things up):
https://gaming-backlog.atlassian.net/browse/GB-74
https://gaming-backlog.atlassian.net/browse/GB-73